spec/ruby/core/numeric/shared/step.rb

```diff
@@ -0,0 +1,409 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
+require File.expand_path('../../fixtures/classes', __FILE__)
+
+describe :numeric_step, :shared => true do
+
+  def get_args(*args)
```

Comment:
I'm not sure if this was already present before (bit hard to grok based on the diff), but if not then this should be moved into a fixture file of sorts. Doing so keeps the spec files clean.